src/audiovideoobserver/AudioVideoObserver.ts

```diff
@@ -33,7 +36,7 @@ export default interface AudioVideoObserver {
   /**
    * Called whenever a tile has been removed.
    */
-  videoTileWasRemoved?(tileId: number): void;
+  videoTileWasRemoved?(tileId: number, status?: VideoTileStatusCode): void;
```

Comment:
Remove the second parameter. See my comment on the`VideoTileStatusCode` file for details.